addons/mass_mailing_sms/models/mailing_list.py

```diff
@@ -27,6 +27,21 @@ def action_view_contacts_sms(self):
         action['context'] = dict(action.get('context', {}), search_default_filter_valid_sms_recipient=1)
         return action
 
+    def action_send_new_sms(self):
+        return {
+            'type': 'ir.actions.act_window',
+            'name': _('Send Sms From Mailing List'),
+            'res_model': 'mailing.mailing',
+            'view_mode': 'form',
+            'views': [(self.env.ref('mass_mailing_sms.mailing_mailing_view_form_sms').id, 'form')],
+            'context': {
+                'default_model_id': self.env.ref('mass_mailing.model_mailing_list').id,
```

Comment:
If necessary, use available getters when accessing ir models, like
```suggestion
                'default_model_id': self.env['ir.model']._get_id('mailing.list'),
```